src/types/team.ts

```diff
@@ -32,6 +33,17 @@ export interface TeamSettings {
   boundary?: string;
 }
 
+export interface GeneralTeamSettings {
```

Comment:
I'd go for the simpler / clearer `TeamSettings` throughout. 

This will simplify things for us as developer because -
 - it matches the database / data model
 - it means we can update the existing `TeamSettings` in `planx-new` instead of maintaining both a `TeamSettings` and `GeneralTeamSettings`
 - "General" is not semantically meaningful enough

I agree that from a user's POV this may not align (e.g. Team Settings > Settings is a bit unclear), but I think we can make this distinction of "General settings" purely at the presentation layer as opposed to the logic layer.